code/src/App.js

```diff
@@ -1,9 +1,37 @@
 import React from 'react'
+import moment from "moment";
+
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+import { GlobalStyle, TodoListContainer } from 'styles'
+
+import tasks from 'reducers/tasks'
+
+import TaskList from 'components/TaskList'
+import AddTask from 'components/AddTask'
+
+const reducer = combineReducers({
+  tasks: tasks.reducer,
+})
+
+const store = configureStore({ reducer })
 
 export const App = () => {
+
+  const date = moment().format("dddd MMMM Do YYYY");
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <GlobalStyle />
+      <Provider store={store}>
+        <TodoListContainer>
+          <p className="date">{date}</p>
+          <h1>PLAN YOUR DAY</h1>
+          <h3>What do you need to do?</h3>
```

Comment:
A suggestion would be to add the header part in a separate component to make the App look a little cleaner :)